string auction_type
string task_name
string task_data

int64 creator_ID
string creator_name
time end_time

float64 min_price
float64 max_price
int64 auction_ID

